docs/reference/cluster/cluster-info.asciidoc

```diff
@@ -238,6 +241,46 @@ Number of failed operations for the processor.
 ======
 
 
+[[cluster-info-api-response-body-threadpool]]
+`thread_pool`::
+(object)
+Contains thread pool information of the cluster.
```

Comment:
Maybe `Contains information about the thread pools of the cluster` reads a bit better? What do you think?